tests/x11/tracker/tracker_by_command.pm

```diff
@@ -24,19 +24,18 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_leap);
 
 sub run {
     x11_start_program('xterm');
     if (is_sle('<12-SP2')) {
         script_run "tracker-search newfile";
     }
     else {
-        script_run 'tracker search emtpyfile';
-        record_soft_failure 'bsc#1074582 tracker can not index empty file automatically' if check_screen 'tracker-cmdsearch-noemptyfile', 30;
-        # Wait 20s for tracker to index the test file
-        wait_still_screen 20;
-        script_run "tracker search newfile";
+        my $trackercmd = (is_sle('<16') or is_leap('<16.0')) ? 'tracker' : 'tracker3';
+        script_run "$trackercmd search emptyfile";
```

Comment:
This needle should match the 'file found' screen, and can be tagged softfailed with the bsc ref on a needle showing the file not found.

Sounds much cleaner than checking screen and 'if not found' report softfail - the needle can happen to not match for a number of reasons (fonts?) and then the softfail is simply lost